feat: [es] Create /docs/contributing/new-content.md (#5270)
open-telemetry/opentelemetry.io: content/es/docs/contributing/new-content.md

Reviewer: I guess that you now know that the page path has changed. Do you want to use the new name as of now?

Reviewer 2: Hmm, was it changes recently? Otherwise, we should use an existing name.

Author: I see is now called pull-requests.md @theletterf

direction TB
    0[ ] -.-
    1[Edita esta página] --> 2[Usa el editor de markdown en Github<br>para hacer tus cambios]
    2 --> 3[agrega tus propuesta de cambios]

end
subgraph tasks2[ ]
direction TB
4[Selecciona tus archivos a cambiar] --> 5[Haz click en Crear Pull Request] --> 6[Llena los cambios en Abriendo un Pull Request]
6 --> 7[Haz click en Crear pull request]
end

id1 --> tasks --> tasks2

classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
class A,1,2,3,4,5,6,7 grey
class 0 spacewhite
class tasks,tasks2 white
class id1 k8s
```

_Figure 2. Pasos para abrir un Pull Request desde Github._

1. En la página de la tarea en Github, selecciona **Editar esta página** en el
   panel de navegación en la derecha.

1. Si no eres miembro de este proyecto, puedes hacer fork al repositorio de
   Github. Selecciona **Fork**.

1. Haz tus cambios desde el editor de Github.

1. Llena el formulario de **Subir cambios**.

> 1. Selecciona en **Proponer cambios**.
> 
> 1. Selecciona en **Crear pull request**.
> 
> 1. Cuando la pantalla de **Abrir un pull request** aparezca. Agrega una
>    descripción para que los supervisores puedan comprender tus cambios.
> 
> 1.Selecciona en **Crear pull request**.

Reviewer 2: ```suggestion
1. Selecciona **Proponer cambios**.

1. Selecciona **Crear pull request**.

1. Cuando la pantalla de **Abrir un pull request** aparezca. Agrega una
   descripción para que los supervisores puedan comprender tus cambios.

1.Selecciona **Crear pull request**.
```

1 Haz click en el botón **Crear pull request**.

Tu pull request estará disponible en
[pull Requests](https://github.com/open-telemetry/opentelemetry.io/pulls).

Después de abrir un PR, GitHub ejecuta pruebas automatizadas e intenta
implementar una vista previa usando [Netlify](https://www.netlify.com/).

- Si la compilación de Netlify falla, seleccione **Detalles** para obtener más
  información.
- Si la compilación de Netlify se realiza correctamente, seleccione **Detalles**
  para abrir una versión preparada del sitio web de OpenTelemetry con los
  cambios aplicados. Así es como los revisores verifican sus cambios.

También pueden fallar otras comprobaciones. Consulte la
[lista de todas las comprobaciones de PR](/docs/contributing/pr-checks).

### Soluciona problemas de contenido automáticamente

Antes de enviar un cambio al repositorio, ejecuta el siguiente comando y (i)
aborda los problemas informados, (ii) confirma los archivos modificados por el
script:

```sh
npm run test-and-fix
```

Para probar y solucionar por separado todos los problemas con tus archivos,
ejecuta:

```sh
npm run test    # Checks but does not update any files
npm run fix:all # May update files
```

> Para enumerar los scripts NPM disponibles, ejecuta `npm run`. Consulta
> [Los pre-requisitos de PR](/docs/contributing/pr-checks) para obtener más
> información sobre las verificaciones de los Pull Request y cómo solucionar
> errores automáticamente.

Reviewer 2: ```suggestion
Para enumerar los scripts NPM disponibles, ejecuta `npm run`. Consulta
los [Pre-requisitos de PR](/docs/contributing/pr-checks) para obtener más
información sobre las verificaciones de los Pull Request y cómo solucionar
errores automáticamente.
```

### Obtén una vista previa de tus cambios localmente {#preview-locally}

Obtén una vista previa de tus cambios localmente antes de enviarlos o abrir un
pull request. Una vista previa te permite detectar errores de compilación o
problemas de formato de Markdown.

> Para crear y servir el sitio localmente con Hugo, ejecute el siguiente comando:

Reviewer 2: ```suggestion
Para crear y servir el sitio localmente con Hugo, ejecuta el siguiente comando:
```

```shell
npm run serve
```

> Navegue a `http://localhost:1313` en su navegador web para ver la vista previa
> local. Hugo observa los cambios y reconstruye el sitio según sea necesario.
> 
> Para detener la instancia local de Hugo, vuelva a la terminal y escriba
> `Ctrl+C`, o cierre la ventana de la terminal.

Reviewer 2: ```suggestion
Navega a `http://localhost:1313` en su navegador web para ver la vista previa
local. Hugo observa los cambios y reconstruye el sitio según sea necesario.

Para detener la instancia local de Hugo, vuelve a la terminal y escriba
`Ctrl+C`, o cierre la ventana de la terminal.
```

### Implementaciones del sitio y vistas previas de PR

> Si envía una PR, Netlify crea una [Vista Previa de Implementación][] para que pueda
> revisar sus cambios. Una vez que se fusiona su PR, Netlify implementa el sitio actualizado
> en el servidor de producción.
> 
> > **Nota**: Las vistas previas de PR incluyen _páginas de borrador_, pero las
> > compilaciones de producción no.
> 
> Para ver los registros de implementación y más, visite el [Panel de Control][]
> del proyecto -- Se requiere inicio de sesión en Netlify.
> 

Reviewer 2: ```suggestion
Si envías una PR, Netlify crea una [Vista Previa de Implementación][] para que puedas
revisar sus cambios. Una vez que se fusiona su PR, Netlify implementa el sitio actualizado
en el servidor de producción.

> **Nota**: Las vistas previas de PR incluyen _páginas de borrador_, pero las
> compilaciones de producción no.

Para ver los registros de implementación y más, visita el [Panel de Control][]
del proyecto. Se requiere inicio de sesión en Netlify.

```

### Pautas de PR

> Antes de fusionar un PR, a veces se requieren algunas iteraciones de revisión y
> edición. Para ayudarnos a nosotros y a usted mismo a hacer que este proceso sea
> lo más sencillo posible, le pedimos que respete lo siguiente:

Reviewer 2: ```suggestion
Antes de fusionar un PR, a veces se requieren algunas iteraciones de revisión y
edición. Para ayudarnos a nosotros y a ti mismo a hacer que este proceso sea
lo más sencillo posible, te pedimos que respetes lo siguiente:
```

> - Si su PR no es una solución rápida, entonces **trabaje desde una copia**: haga
>   clic en el botón
>   [Fork](https://github.com/open-telemetry/opentelemetry.io/fork) en la parte
>   superior del repositorio y clone el repositorio localmente. Cuando esté listo,
>   genere un PR al repositorio original.
> - **No trabaje desde la rama `main`** de su fork, sino que cree una rama
>   específica para el PR.
> - Asegúrate de que los mantenedores tengan permiso para aplicar cambios a tu
>   solicitud de incorporación de cambios
>   (https://docs.github.com/es/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

Reviewer 2: ```suggestion
- Si su PR no es una solución rápida, entonces **trabaja desde una copia**: 
  selecciona el botón
  [Fork](https://github.com/open-telemetry/opentelemetry.io/fork) en la parte
  superior del repositorio y clone el repositorio localmente. Cuando esté listo,
  genere un PR al repositorio original.
- **No trabajes desde la rama `main`** de su fork: crea una rama
  específica para el PR.
- Asegúrate de que los mantenedores tengan permiso para aplicar cambios a tu
  solicitud de incorporación de cambios
  (https://docs.github.com/es/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
```

### Cambios de los revisores

> A veces, los revisores se suben cambios en tu pull request. Antes de realizar
> cualquier otro cambio, actualiza tu rama.

Reviewer 2: ```suggestion
A veces, los revisores suben cambios en tu pull request. Antes de realizar
cualquier otro cambio, actualiza tu rama.
```

1. Actualiza los commits de tu fork y haz rebase de tu rama actual:

   ```shell
   git fetch origin
   git rebase origin/<nombre-de-tu-rama>
   ```

1. Luego de hacer rebase, sube tus nuevos cambios a tu fork:

   ```shell
   git push --force-with-lease origin <nombre-de-tu-rama>
   ```

También puedes resolver conflictos de fusión desde la interfaz de usuario de
GitHub.

### Conflictos en ramas y git rebase

Si otro colaborador realiza cambios en el mismo archivo en otro pull request,

> puede crear un merge. Es tu responsabilidad resolver todos los conflictos en tu

Reviewer 2: ```suggestion
puedes crear un merge. Es tu responsabilidad resolver todos los conflictos en tu
```

Pull Request.

1. Actualiza tu fork y haz git rebase de tu rama local:

   ```shell
   git fetch origin
   git rebase origin/<nombre-de-tu-rama>
   ```

   Luego haz force-push de tus cambios a tu fork:

   ```shell
   git push --force-with-lease origin <nombre-de-tu-rama>
   ```

1. Actualiza los cambios de la rama `upstream/main` en
   `open-telemetry/opentelemetry.io` y haz git rebase a tu rama actual:

   ```shell
   git fetch upstream
   git rebase upstream/main
   ```

1. Inspecciona los resultados del git rebase:

   ```shell
   git status
   ```

   Esto indicará la cantidad de archivos marcados con conflictos.

1. Abra cada archivo en conflicto y busque los marcadores de conflicto: `>>>`,
   `<<<`, y `===`. Resuelva el conflicto y elimine el marcador de conflicto.

> Para obtener más información, consulte

Reviewer 2: ```suggestion
Para obtener más información, consulta
```

[Cómo se presentan los conflictos](https://git-scm.com/docs/git-merge#_how_conflicts_are_presented).

> 1. Agregue los archivos al conjunto de cambios:
> 
>    ```shell
>    git add <filename>
>    ```
> 
> 1. Continúe con el rebase:
> 
>    ```shell
>    git rebase --continue
>    ```
> 
> 1. Repita los pasos 2 a 5 según sea necesario.
> 
>    Después de aplicar todas las confirmaciones, el comando `git status` muestra
>    que el rebase está completo.

Reviewer 2: ```suggestion
1. Agrega los archivos al conjunto de cambios:

   ```shell
   git add <filename>
   ```

1. Continúa con el rebase:

   ```shell
   git rebase --continue
   ```

1. Repite los pasos 2 a 5 según sea necesario.

   Después de aplicar todas las confirmaciones, el comando `git status` muestra
   que el rebase está completo.
```
